ecl/eclcc/eclcc.cpp

```diff
@@ -2146,6 +2151,30 @@ void EclCompileInstance::getTargetPlatform(StringBuffer & result)
         return eclcc.getTargetPlatform(result);
 }
 
+void EclCC::appendNeverSimplifyList(const char *attribsList)
+{
+    const char * p = attribsList;
+    while (*p)
+    {
+        StringBuffer attribRegex;
+        if (*p == ',') p++;
+        for (; *p && *p != ','; p++)
+        {
+            if (*p=='/' || *p=='.' || *p=='\\')
+                attribRegex.append("\\.");
+            else
```

Comment:
It would be useful to map '?' to '.' and '*' to '.*' so that you can use wildcards to include all attributes in a module.  E.g. "ut/*".

Reply:
You've mentioned wildcards are not necessary as all sub-scopes will be matched automatically (so no changes here).